blueocean-web/src/main/js/try.js

```diff
@@ -0,0 +1,18 @@
+var $ = require('jquery-detached').getJQuery();
+var jsModules = require('@jenkins-cd/js-modules');
+
+$(document).ready(() => {
+    var tryBlueOcean = $('<div class="try-blueocean header-callout">Try Blue Ocean UI ...</div>');
```

Comment:
The `<a href="">` code would be the same anyway, seeing as you're going to need JavaScript to get that element into the right place in the DOM.